Libraries/Text/RCTShadowText.m

```diff
@@ -278,7 +278,7 @@ - (void)_setParagraphStyleOnAttributedString:(NSMutableAttributedString *)attrib
   [attributedString enumerateAttribute:NSParagraphStyleAttributeName inRange:(NSRange){0, attributedString.length} options:0 usingBlock:^(id value, NSRange range, BOOL *stop) {
     if (value) {
       NSParagraphStyle *paragraphStyle = (NSParagraphStyle *)value;
-      CGFloat maximumLineHeight = round(paragraphStyle.maximumLineHeight / self.fontSizeMultiplier);
+      CGFloat maximumLineHeight = round(paragraphStyle.maximumLineHeight / (self.fontSizeMultiplier > 0.0 ? self.fontSizeMultiplier : 1.0));
```

Comment:
It would probably be better to identify how/where self.fontSizeMultiplier can end up getting set to zero, and prevent it at that point.